src/libcec.cpp

```diff
@@ -51,39 +51,34 @@ const map<enum cec_user_control_code, const char *> Cec::cecUserControlCodeName
 // We store a global handle, so we can use g_cec->ToString(..) in certain cases. This is a bit of a HACK :(
 static ICECAdapter * g_cec = NULL;
 
-int cecLogMessage(void *cbParam, const cec_log_message message) {
+void cecLogMessage(void *cbParam, const cec_log_message *message) {
```

Comment:
In libcec.h below this is defined as `friend void cecLogMessage (void *cbParam, const CEC::cec_log_message &message);` which doesn't match this (or in-fact the previous signature). We should fix that.

Reply:
Updated